real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -36,7 +40,7 @@
 	// this program is started
 	public static boolean ACTIVE_DEFAULT = false;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1795141970/issues/source?bid=2938665&fileBranchId=3059924#l53)